lib/matplotlib/backend_managers.py

```diff
@@ -43,11 +43,11 @@ class ToolManager:
 
     Attributes
     ----------
-    figure : `Figure`
-    keypresslock : `widgets.LockDraw`
-        `LockDraw` object to know if the `canvas` key_press_event is locked
-    messagelock : `widgets.LockDraw`
-        `LockDraw` object to know if the message is available to write
+    figure : `~matplotlib.figure.Figure`
```

Comment:
Lets stick to just \`.Figure\`?  (other objects may be more obscure and thus fully qualified, but .Figure doesn't need the full module name...)

Reply:
That's not what our docstring guide says https://matplotlib.org/devdocs/devel/documenting_mpl.html#referencing-types. (But hasn't been consistent throughout the file anyway. So, reverted for now.)

Reply:
I could rather change the guide in that case...